website/src/components/Chat/ChatConversation.tsx

```diff
@@ -200,6 +200,22 @@ export const ChatConversation = memo(function ChatConversation({ chatId, getConf
     [createAndFetchAssistantMessage, isSending, setIsSending]
   );
 
+  const messagesEndRef = useRef<HTMLDivElement>(null);
+  const chatContainerRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (!chatContainerRef.current || !messagesEndRef.current) return;
+    const { scrollTop, scrollHeight, clientHeight } = chatContainerRef.current;
+    const threshold = 100;
```

Comment:
This is a magic number, I tested on different screen widths and found this is the most optimal one lol

Reply:
We can just always scroll to the bottom, OpenAI does it?

Reply:
Yes, the lib they use already handles that case.